cobra/flux_analysis/deletion.py

```diff
@@ -341,38 +351,43 @@ def double_reaction_deletion(model,
 def double_gene_deletion(model, gene_list1=None, gene_list2=None, **kwargs):
     """Sequentially knocks out pairs of genes in a model
 
-    model : :class:`~cobra.core.Model.Model`
-        cobra model in which to perform deletions
+    Parameters
+    ----------
+    model : cobra.Model
+        The metabolic model to perform deletions in.
 
-    gene_list1 : [:class:`~cobra.core.Gene.Gene`:] (or their id's)
-        Genes to be deleted. These will be the keys in the resulting dict.
-        If not provided, all genes will be used.
+    gene_list1 : iterable, optional
+        First iterable of `cobra.Gene`s to be deleted. If not passed,
+        all the genes from the model are used.
 
-    gene_list1 : [:class:`~cobra.core.Gene.Gene`:] (or their id's)
-        Genes to be deleted. These will be the keys in the resulting dict.
-        If not provided, gene_list1 will be used.
+    gene_list2 : iterable, optional
+        Second iterable of `cobra.Gene`s to be deleted. If not passed,
+        all the genes from the model are used. The product of two
+        genes lists will be used to perform the knockouts.
+
+    method: {"fba", "moma"}, optional
+        Procedure used to predict the growth rate.
 
-    method: "fba" or "moma"
-        Procedure used to predict the growth rate
+    number_of_processes : int, optional
+        Number of parallel processes to run. Can speed up the computations
+        if number of knockouts to perform is large. If not passed,
+        will be set to the number of CPUs
 
-    solver: str for solver name
+    solver: str, optional
         This must be a QP-capable solver for MOMA. If left unspecified,
         a suitable solver will be automatically chosen.
 
-    zero_cutoff: float
-        When checking to see if a value is 0, this threshold is used.
-
-    number_of_processes: int for number of processes to use.
-        If unspecified, the number of parallel processes to use will be
-        automatically determined. Setting this to 1 explicitly disables used
-        of the multiprocessing library.
-
-    .. note:: multiprocessing is not supported with method=moma
```

Comment:
Why not? There is nothing particular in moma that would inhibit that. You can even run `add_moma` in the master process and send the model afterwards. Also that note is absent for `double_reaction_deletion`.